### PR TITLE
update hammerdb template

### PR DESCRIPTION
**Pull request by ebattat**
Fix:

Add ability to compare between 2 different databases

### CODE REVIEW

File: benchmark_runner/jupyterlab/templates/analyze_workloads/analyze_hammerdb_logs/analyze_hammerdb_logs.ipynb [hunk 1]
```diff
@@ -35,10 +35,10 @@
    "outputs": [],
    "source": [
     "# S3_LOGS_URLS = ['412URL', '413URL'] - for comparing 2 artifacts \n",
-    "S3_LOGS_URLS = ['']\n",
+    "S3_LOGS_URLS = ['', '']\n",
     "# Compare versions/ database\n",
-    "versions = ['412', '413']\n",
-    "database = 'Mariadb'"
+    "versions = ['4.12', '4.13']\n",
+    "database = ['PostgreSQL', 'PostgreSQL']"
```
[the remark this answers is not in the document]
Comment (RobertKrawitz) [reply benchmark_runner/jupyterlab/templates/analyze_workloads/analyze_hammerdb_logs/analyze_hammerdb_logs.ipynb:41]: That's consistent with the code below, but I am curious why two databases when they will always be the same (we don't want to compare MariaDB with PostgreSQL).

[the remark this answers is not in the document]
Comment (ebattat) [reply benchmark_runner/jupyterlab/templates/analyze_workloads/analyze_hammerdb_logs/analyze_hammerdb_logs.ipynb:41]: Why not, it should be flexible compare

[the remark this answers is not in the document]
Comment (RobertKrawitz) [reply benchmark_runner/jupyterlab/templates/analyze_workloads/analyze_hammerdb_logs/analyze_hammerdb_logs.ipynb:41]: Compare between what?  Two different databases?

[the remark this answers is not in the document]
Comment (ebattat) [reply benchmark_runner/jupyterlab/templates/analyze_workloads/analyze_hammerdb_logs/analyze_hammerdb_logs.ipynb:41]: Yes, its only an option, not something we are doing.

[the remark this answers is not in the document]
Comment (RobertKrawitz) [reply benchmark_runner/jupyterlab/templates/analyze_workloads/analyze_hammerdb_logs/analyze_hammerdb_logs.ipynb:41]: OK, this is a reasonable explanation.

**Review by RobertKrawitz — APPROVED** [review]

**Review by athiruma — APPROVED** [review]
/lgtm

**Pull request merged**